change/@fluentui-react-charting-2020-11-05-12-29-32-user-v-jasha-GBarChartCherryPick.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
should this be patch, minor or major?

Reply:
Hi raghu,

All other PR's (which were cherry-picked and ported to master) are with type:none only. And while adding change file they are only 2 options(Prerelease and None). So I have added None.